docs/pages/docs/query/client.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Client"
+description: "Query a Ponder app with the `@ponder/client` package."
+---
+
+import { Callout, Steps, Tabs } from "nextra/components";
+
+# `@ponder/client`
+
+The `@ponder/client` package is a TypeScript client for querying a Ponder app with end-to-end type safety.
+
+## Example projects
+
+These example apps demonstrate how to use `@ponder/client`.
+
+- [**Basic**](https://github.com/ponder-sh/ponder/blob/kjs/client-db/examples/with-client/client/src/index.ts)
+- [**NextJs**](https://github.com/ponder-sh/ponder/blob/kjs/client-db/examples/with-nextjs/frontend/src/hooks/useDeposits.ts)
+
+## Get started
+
+<Steps>
+
+### Update to `>=0.9.0`
+
+`@ponder/client` is available starting from version `0.9.0`. Read the [migration guide](/docs/migration-guide#090) for more details.
+
+### Register client middleware
+
+```ts filename="src/api/index.ts" {7}
+import { db } from "ponder:api";
+import { Hono } from "hono";
+import { client } from "ponder";
+
+const app = new Hono();
+
+app.use(client({ db }));
+
+export default app;
+```
+
+### Install `@ponder/client`
+
+{/* prettier-ignore */}
+<Tabs items={["pnpm", "yarn", "npm"]}>
+  <Tabs.Tab>
+    ```bash filename="shell"
+    pnpm add @ponder/client
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash filename="shell"
+    yarn add @ponder/client
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash filename="shell"
+    npm add @ponder/client
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+### Create a client and send a request
+
+```ts {4,6}
+import { createClient } from "@ponder/client";
+import * as schema from "../../ponder/ponder.schema";
+
+const client = createClient("https://...", { schema });
```

Comment:
Should make it more clear what HTTP URL you need to pass here - root path of the ponder server?